src/components/segmented-control/segmented-control-styles.jsx

```diff
@@ -52,12 +52,13 @@ export default theme => {
       },
 
       '&.secondary': {
+        letterSpacing: 0.2,
         textAlign: 'center',
         outline: 'none',
         fontSize: '12px',
         fontFamily: typography.primary.fontFamily,
         fontWeight: typography.fontWeightSemiBold,
-        color: palette.color.gray,
+        color: palette.color.grayDark,
```

Comment:
Can you change this to `palette.text.muted`? (It is grayDark - https://github.com/nordnet/nordnet-ui-kit/blob/master/src/styles/palette.js#L15)